controllers/bootstrap/init.go

```diff
@@ -464,9 +464,16 @@ func (b *Bootstrap) installNssOperator(manualManagement bool) error {
 		return err
 	}
 
+	// Create General NSS CRs
 	if err := b.renderTemplate(constant.NamespaceScopeCR, b.CSData); err != nil {
 		return err
 	}
+	// Create NSS CRs managedby ODLM for Single CS instance case
+	if b.CSData.MasterNs == b.CSData.ControlNs {
```

Comment:
Is my understanding correct? I believe ControlNs is empty ("") in the on-prem with single CS.

Reply:
When we first create the `bootstrap`, the `ControalNs` is empty.

But at the `initialResource`, we will assign `MasterNs` to `ControlNs` if `ControlNs` is empty, for the convenience to deploy `webhook` and `secretshare` in both Single and Multi cases.
https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/bootstrap/init.go#L165
https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/bootstrap/init.go#L198

Reply:
Thanks for the explanation! I understand.